source/common/microkernel_architecture/microkernel_modules/network_manager/ANetworkManager.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef ANETWORK_MANAGER_HPP_
+# define ANETWORK_MANAGER_HPP_
+
+# include "AFunctionality.hpp"
+# include "InstructionBuffer.hpp"
+# include <QDebug>
+
+class ANetworkManager : public AFunctionality
+{
+    Q_OBJECT
+
+public:
+    ANetworkManager(bool threaded);
+    virtual ~ANetworkManager();
+
+protected:
+    template<class Socket>
+    bool _read(InstructionBuffer *instruction, Socket &socket, qint64 &bytesAvailable)
+    {
+        qint64 readSize = -1, ret = -1;
```

Comment:
@BernardJeremy it looks like we haven't the same indentation...
` source/server/microkernel_architecture/microkernel_modules/database_manager/Entity.hpp`
100th line